### ISSUE TITLE
closing a <track> element causes compilation to fail

### ISSUE THREAD

**Issue by oliverlangan** [tildeio/htmlbars#391]
When using a <track> within a <video> element, if it is not closed HTMLBars will correctly compile; but if it is closed with a </track> then HTMLBars fails within the EngTag block.

"Error: Closing tag `track` (on line 5) did not match last open tag `video` (on line 3)."

This is unfortunate, because WebKit (Safari and Chrome both) appends the closing tag when getting the .outerHTML from the DOM.

Sample of failing HTML snippet:
<video height="200" width="470" poster="Art/sintel.jpg" crossorigin="anonymous" controls="">
<source src="Art/sintel.mp4" type="video/mp4">
<track src="Art/sintel-captions.vtt" kind="captions" label="Closed Captions" default=""></track>
</video>

(adapted from http://www.jwplayer.com/html5/track/ )

**Comment by mmun** [comment]
Track is a void element so this behaviour is "expected". We could provide a better error message.

It sounds like a bug if `outerHTML` behaves that way. Is it relevant, though?

**Comment by oliverlangan** [comment]
I agree this feels like a bug in WebKit's outerHTML. Firefox behaves correctly.
https://bugs.webkit.org/show_bug.cgi?id=147084
